extensions/kubeapi/rbac/rbac.go

```diff
@@ -66,3 +72,32 @@ var RoleTemplateGroupVersionResource = schema.GroupVersionResource{
 	Version:  Version,
 	Resource: "roletemplates",
 }
+
+// ProjectRoleTemplateBindingGroupVersionResource is the required Group Version Resource for accessing projectroletemplatebindings in a cluster, using the dynamic client.
+var ProjectRoleTemplateBindingGroupVersionResource = schema.GroupVersionResource{
+	Group:    GroupName,
+	Version:  Version,
+	Resource: "projectroletemplatebindings",
+}
+
+
+// GetProjectRoleTemplateBindingsByName is a helper function that uses the dynamic client to get a projectroletemplatebinding from local cluster.
+func GetProjectRoleTemplateBindingsByName(client *rancher.Client, getOpts metav1.GetOptions, name string) (*v3.ProjectRoleTemplateBinding, error) {
```

Comment:
Since Public API was introduced in 2.8, can you remove this for 2.7?

Reply:
Do you still want this Priya? Should we wait on this PR until that is removed?

Reply:
Israel, I think we should wait until this is removed for 2.7